quaddtype/quaddtype/src/umath.c

```diff
@@ -65,9 +65,10 @@ quad_multiply_resolve_descriptors(PyObject *self, PyArray_DTypeMeta *dtypes[],
     Py_INCREF(given_descrs[1]);
     loop_descrs[1] = given_descrs[1];
 
-    Py_INCREF(given_descrs[1]);
-    loop_descrs[1] = given_descrs[1];
-    return NPY_SAFE_CASTING;
+    // For now, assume that the input dtypes are all simply quads
```

Comment:
loop_descrs[2] is the output dtype

Reply:
Thanks, I added a little more explanation of what I meant by this so it is no longer ambiguous.